README.md

```diff
@@ -157,7 +178,28 @@ RIOT closely collaborates with and inspires other open source projects, e.g., [A
 
 ## Contribute
 
-To contribute something to RIOT, please refer to our
+We welcome contributors of all experience levels! Here’s how you can help:
+
+### 📌 Quickstart Guide for New Contributors  
+1. **Fork & Clone** the repository:
+   Install required dependencies (compiler, build tools)
+   ```sh
+   git clone https://github.com/YOUR-USERNAME/RIOT.git
+   cd RIOT
+2. Setup Development Environment
+   Test your exam
+   ```sh
+   cd examples/hello-world
```

Comment:
After a recent changes in examples directorie - hello-word is now in `examples/essentials/hello-world`.